### PR TITLE
Majid/fix/links favorite series

### PR DESCRIPTION
**Pull request by mhadaily**
## Description

Just cleaning up stuff to address Chris's points 

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [ x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ x] All existing and new tests are passing.
- [ x] I updated/added relevant documentation (doc comments with `///`).
- [ x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [ x] I read and followed the [Flutter Style Guide].
- [ x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [ x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/fluttercommunity/plus_plugins/blob/main/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning

### CODE REVIEW

File: packages/android_alarm_manager_plus/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 1.0.1
+
+ - **REFACTOR**: rename e2e test files to _plus_e2e to be compatible with melos (#40).
```
Comment (miquelbeltran) [inline packages/android_alarm_manager_plus/CHANGELOG.md:3]: I think we don't really need a new release for this

File: packages/android_intent_plus/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 1.0.1
+
+ - **REFACTOR**: rename e2e test files to _plus_e2e to be compatible with melos (#40).
```
Comment (miquelbeltran) [inline packages/android_intent_plus/CHANGELOG.md:3]: same.

File: packages/battery_plus/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+## 1.0.1
+
+ - **REFACTOR**: rename e2e test files to _plus_e2e to be compatible with melos (#40).
+ - **REFACTOR**: move e2e tests to example/test_driver and update docs (#39).
```
Comment (miquelbeltran) [inline packages/battery_plus/CHANGELOG.md:4]: same.

**Review by miquelbeltran — APPROVED** [review]
Thanks! I don't think that internal changes on the e2e testing need to be in the changelog or require a release but I may be wrong.

File: packages/android_alarm_manager_plus/CHANGELOG.md (hunk 1 above)
Comment (mhadaily) [reply packages/android_alarm_manager_plus/CHANGELOG.md:3]: I absolutely agree and I have no idea how this came to my PR! I have not changed this at all!

**Comment by mhadaily** [comment]
@miquelbeltran I think it should be fine now, there was a local commit on my `main` branch so when I branched out it came to this PR. I fixed it.

**Review by miquelbeltran — APPROVED** [review]
Thanks!

**Comment by mhadaily** [comment]
@miquelbeltran  should we merge or wait to fix CD/CI first?

**Comment by miquelbeltran** [comment]
I don't know, I am not sure why this dependency fails:

```
 [battery_plus_linux]: Deletion failed, path = '/home/runner/.pub-cache/hosted/pub.dartlang.org/characters-1.1.0' (OS Error: No such file or directory, errno = 2)
```

**Comment by mhadaily** [comment]
mm, might be cache or maybe @jpnurmi  knows? do you JP?

**Comment by miquelbeltran** [comment]
merge main, we increased the timeout and that should help

**Pull request merged**